src/go_attack/baseline_attack.py

```diff
@@ -249,6 +251,7 @@ def run_baseline_attack(
     model_path: Optional[Path] = None,
     num_visits: Optional[int] = None,
     passing_behavior: Optional[str] = None,
+    victim: Literal["B", "W"] = "B",
```

Comment:
For consistency with other uses
```suggestion
    victim_color: Literal["B", "W"] = "B",
```

Reply:
`victim_color` is already defined as `Color.from_str(victim)` in the body of this function, and I think `victim_color = Color.from_str(victim_color)` would be confusing due to mixing types. I'll rename to `victim_color_str` in both this function and `make_log_dir`

Reply:
ah makes sense, `victim_color_str` seems reasonable